packages/core/src/framebuffer/Framebuffer.js

```diff
@@ -32,6 +32,14 @@ export class Framebuffer
 
         this.glFramebuffers = {};
 
+        /**
+         * Asks framebuffer to use multi-sampling if available, 0 by default.
+         * See {@link PIXI.MSAA_QUALITY}
+         * @member {number}
+         * @default PIXI.MSAA_QUALITY.LOW
```

Comment:
Why is the "default" `multisample` property `MSAA_QUALITY.LOW` and not `NONE`? I think the latter makes more sense.

Reply:
Oops, should be NONE